### DISCUSSION TITLE
SRCCLR-SID-30177 - self answered

### DISCUSSION THREAD

**Discussion by CobyPear** [i18next/i18next#1628] [Q&A]
This vulnerability was flagged by a Veracode/Sourceclear scan and I couldn't find any information about it. I was wondering if it had been addressed somewhere or if it's some kind of false positive?

Link to Veracode page: https://www.sourceclear.com/vulnerability-database/security/prototype-pollution/javascript/sid-30177/summary

Edit:
I see I was looking on the wrong page and it looks like the latest npm version of the package has addressed this!

**Comment by jamuhl** [comment]
Yes...was fixed a while ago.
